chunky/src/java/se/llbit/chunky/chunk/biome/GenericQuartBiomeData3d.java

```diff
@@ -0,0 +1,47 @@
+package se.llbit.chunky.chunk.biome;
+
+import it.unimi.dsi.fastutil.ints.Int2ObjectOpenHashMap;
+
+import static se.llbit.chunky.world.Chunk.*;
+import static se.llbit.chunky.world.Chunk.Z_MAX;
+
+/**
+ * Implementation of a
```

Comment:
of a 3D biome grid where the smallest size of a biome is 4x4x4 blocks, as used since Minecraft 1.15?

Reply:
wait did I just not comment this... what :facepalm: